src/ReportGenerator.Core/Parser/DynamicCodeCoverageParser.cs

```diff
@@ -187,7 +187,9 @@ private static CodeFile ProcessFile(XElement module, string fileId, ClassWithNam
             var linesOfFile = methods
                 .Elements("ranges")
                 .Elements("range")
+                .Where(r => r.Attribute("source_id").Value == fileId)
                 .Where(l => l.Attribute("start_line").Value != "15732480")
+                .Distinct()
```

Comment:
Does `Distinct()` make any difference here? You apply `Distinct()` to a list of _XElement_ objects, this will always return the full list of elements. I guess you can remove that line.

Reply:
Will remove it.